scene/main/node.cpp

```diff
@@ -2974,9 +2974,19 @@ static void _print_orphan_nodes_routine(Object *p_obj) {
 		path = String(p->get_name()) + "/" + p->get_path_to(n);
 	}
 
+	String source;
+	Variant script = n->get_script();
+	if (!script.is_null()) {
```

Comment:
You can assign it directly:
```suggestion
	Ref<Script> script = n->get_script();
	if (script.is_valid()) {
```

Reply:
This change has been made.